README.md

```diff
@@ -130,9 +136,9 @@ cp ${xtt_root_directory}/examples/data/client/* .
 ```
 
 The client executable can take the IP and port of the server as parameters
-(run `xtt_client -h` for a full help on all available parameters):
+(run `./xtt runclient -h` for a full help on all available parameters):
 ```bash
-xtt_client -a 127.0.0.1 -p 4444
+./xtt runclient -a 127.0.0.1 -p 4444
```

Comment:
We need to include the command line options to tell it where the root cert is, right? Something like:
```
xtt runclient -a localhost -p 4444 -e <path-to-root-cert>
```